frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -258,6 +258,12 @@ entitydetails-ContextIssueButton--context-issue-button = REQUEST CONTEXT or REPO
 entitydetails-GenericOriginalString--plural = PLURAL
 entitydetails-GenericOriginalString--singular = SINGULAR
 
+## Entities List
+## Show a list of entities
```

Comment:
This section refers to a component (as opposed to a module), so we should tweak the comments a little bit:

```
## Entities List Entity
## Displays a single Entity as a list element
```

Nit: Please also make sure there are 2 blank lines between this and the previous section (instead of just 1).